Makefile

```diff
@@ -1098,12 +1098,8 @@ release-alias-tag: ## Add the release alias tag to the last build tag
 	gcloud container images add-tag $(TEST_EXTENSION_IMG):$(TAG) $(TEST_EXTENSION_IMG):$(RELEASE_ALIAS_TAG)
 
 .PHONY: release-notes
-release-notes: $(RELEASE_NOTES_DIR) $(RELEASE_NOTES)
-	if [ -n "${PRE_RELEASE}" ]; then \
-	echo ":rotating_light: This is a RELEASE CANDIDATE. Use it only for testing purposes. If you find any bugs, file an [issue](https://github.com/kubernetes-sigs/cluster-api/issues/new)." > $(RELEASE_NOTES_DIR)/$(RELEASE_TAG).md; \
-	else \
-	go run ./hack/tools/release/notes.go --from=$(PREVIOUS_TAG) > $(RELEASE_NOTES_DIR)/$(RELEASE_TAG).md; \
-	fi
+release-notes:
```

Comment:
We should probably update this target to be `release-notes` tool, as it's confusing to have this target just build the tool rather than producing the notes themselves.

Reply:
@killianmuldoon the target is already named as you are suggesting, perhaps you meant something else?

Reply:
pushed up some changes that i think resolve this -- lmk if i misunderstood the recommendation